src/jetson/AES.py

```diff
@@ -8,23 +8,39 @@
 THRES = 400
 
 class encryption():
```

Comment:
Please change this to `Encryption` instead of `encryption` so it follows the [pep8](https://www.python.org/dev/peps/pep-0008/#class-names) python style guide